src/App.js

```diff
@@ -388,12 +388,15 @@ function App() {
     script.id = 'googleMaps';
     document.body.appendChild(script);
     const parameters = getParameters();
+    parameters.bounds = parameters.bounds?.replaceAll('!',',');
     const map = new Map({
       onLoad: loaded,
       onClickTree: showPanel,
       onFindNearestAt: handleFindNearestAt,
       onError: handleError,
       filters: parameters,
+      tileServerUrl: process.env.REACT_APP_TILE_SERVER_URL,
+      tileServerSubdomains: process.env.REACT_APP_TILE_SERVER_SUBDOMAINS.split(","),
```

Comment:
if the Map from the master branch is kept unchanged, this is not required

Reply:
done